src/server/middleware/proxyMiddleware.ts

```diff
@@ -56,13 +56,22 @@ const staticAssetsMiddleware = createProxyMiddleware('/static', {
   target: 'http://localhost:8081'
 });
 
-const apiProxyMiddleware = createProxyMiddleware('/api', {
+const apiProxyMiddleware = createProxyMiddleware(['/api/**', '!/api/thoas'], {
   target: 'https://staging-2020.ensembl.org',
   changeOrigin: true,
   secure: false
 });
 
-let proxyMiddleware = [apiProxyMiddleware];
+const thoasProxyMiddleware = createProxyMiddleware('/api/thoas', {
+  target: 'http://map-thoas.review.ensembl.org',
+  pathRewrite: {
+    '^/api/docs': '/api' // rewrite path
```

Comment:
Just wanted to check if this is in the right place and not supposed to be under `apiProxyMiddleware`?

Reply:
The changes in this file will need to be reverted before the merge.

Reply:
Reverted the changes; ready for the merge; waiting the backend to fix thoas data for non-vertebrates